### PR TITLE
fix: remove the console override

### PR DESCRIPTION
**Pull request by farfromrefug**
Add a note in the Readme on how to bring it back at the app level

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -85,6 +85,38 @@ A huge thank you for all of the backers / sponsors!
 
 This plugin integrates [Vue](https://vuejs.org/) and [NativeScript](https://www.nativescript.org/), allowing you to build cross-platform iOS and Android apps using Vue.
 
+
+# console.log
```
Comment (rigor789) [inline README.md:89]: Let's move this to a separate md file - `CONSOLE_LOG_OVERRIDE.md` and we will link to it from the changelog's `BREAKING CHANGE` section.

File: README.md [hunk 2]
```diff
@@ -85,6 +85,38 @@ A huge thank you for all of the backers / sponsors!
...
+
+In previous versions of `nativescript-vue` the `console.log` was overriden to bring colors to your console.
+It was remove to improve perfomances and reduce the size of the plugin.
+You can however bring it back at an application level.
+Keep in mind that this override should not be present in production. It could make your app slower.
+```
+npm install util-inspect
```
Comment (rigor789) [inline README.md:96]: Perhaps add `--save-dev`

File: README.md [hunk 3]
```diff
@@ -85,6 +85,38 @@ A huge thank you for all of the backers / sponsors!
...
+      ...Array.prototype.map.call(args, function(arg) {
+        return inspect(arg, {
+          depth: 2,
+          colors: Vue.config.debug,
+          showHidden: true
+        }).replace(newLineRegExp, '\n')
+      })
+    )
+  }
+})(console.log, inspect, Vue)
```
Comment (rigor789) [inline README.md:117]: Let's wrap this in an if statement `TNS_ENV !== 'production'` - not sure if the import works in a if block - so perhaps use `require` instead?

Comment (farfromrefug) [reply README.md]: done

**Pull request merged**